### PR TITLE
Removed reference to releases before v1.0.0 and added headings for added changes

### PR DESCRIPTION
**Pull request by pericsson**
Updated the changelog so that duplicate releases before v1.0.0 are removed; fixes nf-core/smrnaseq#37

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/smrnaseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/smrnaseq)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/smrnaseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,12 +41,14 @@
 * Added `conda-forge::pygments=2.5.2`
 * Removed `conda-forge::r-markdown=1.0`
 
-## v1.0.0 - 2019-09-19
+## [v1.0.0] - 2019-09-19
```
Comment (ewels) [inline CHANGELOG.md:44]: Whilst valid, I prefer inline links for nearly all use cases in Markdown. It's the style that we tend to use with nf-core and personally I find it easier to keep track of / maintain.

```suggestion
## [v1.0.0](https://github.com/nf-core/smrnaseq/releases/tag/1.0.0) - 2019-09-19
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -74,15 +76,29 @@
...
-* Software updates: trim-galore 0.4.5 to 0.5.0; samtools 1.8 to 1.9; multiqc 1.5 to 1.6.
+* Option for sequencing centre in BAM file.
+
+### Software updates
+
+* trim-galore 0.4.5 -> 0.5.0
+* samtools 1.8 -> 1.9
+* multiqc 1.5 -> 1.6
+
+[v1.0.0]: https://github.com/nf-core/smrnaseq/releases/tag/1.0.0
```
Comment (ewels) [inline CHANGELOG.md:104]: See above comment

```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -74,15 +76,29 @@
...
-* Add miRTrace in the pipeline.
-* Software updates: multiqc 1.6 to 1.7.
+* multiqc 1.6 -> 1.7.
 
-## 1.0 - 2018-08-06
+## 2018-08-06
+
+### Added
 
 * Switch from SciLifeLab/NGI-smRNAseq to nf-core/smrnaseq.
```
Comment (ewels) [inline CHANGELOG.md:94]:
```suggestion
* Port original pipeline [SciLifeLab/NGI-smRNAseq](https://github.com/SciLifeLab/NGI-smRNAseq) to [nf-core/smrnaseq](https://github.com/nf-core/smrnaseq).
```

**Review by ewels — APPROVED** [review]
Couple of tiny nitpicks, but otherwise looks great! 👍🏻

File: CHANGELOG.md (hunk 1 above)
Comment (pericsson) [reply CHANGELOG.md]: I agree, that looks cleaner!

**Comment by ewels** [comment]
Thanks! 🚀

**Pull request merged**